src/Components/Theme/theme.ts

```diff
@@ -80,64 +94,44 @@ const ALLOWED_STYLES = [
 	'--selected-grid',
 ];
 
+// prettier-ignore
+const CATEGORIES = [
+	'root',
+	'windowmanager',
+	'tabs',
+	'settings',
+	'favorites',
+	'grid',
+	'contextmenu',
+	'prompt',
+	'preview',
+	'properties',
+];
+
 /**
  * Change page theme
  * @param {string} theme - The current theme
  * @returns {Promise<void>}
  */
-const changeTheme = async (theme?: string, category?: '*' | 'root' | 'tabbing' | 'favorites' | 'grid'): Promise<void> => {
+const changeTheme = async (theme?: string, category?: Category): Promise<void> => {
 	if (!category) category = '*';
-	const appearance = await Storage.get('appearance');
+	const appearance = (await Storage.get('appearance')) || {};
 	if (category === '*' || category === 'root') {
-		document.body.style.setProperty('--edge-radius', appearance?.frameStyle === 'os' || !isTauri ? '0px' : '10px');
-		document.body.style.fontSize = appearance?.fontSize ?? '16px';
-		document.documentElement.style.fontSize = appearance?.fontSize ?? '16px';
-		document.body.style.fontFamily = appearance?.fontFamily ?? 'system-ui';
-		document.documentElement.style.fontFamily = appearance?.fontFamily ?? 'system-ui';
```

Comment:
@kimlimjustin Why do you set properties for both `body` and `documentElement`? Aren't they the same, or am I missing anything?

Reply:
For some reason that sometimes it doesn't inherit the style from `body`, so I made it on `documentElement` too, also, I think that `--edge-radius`  shouldn't be removed as it's being used too